tests/test_upgrading.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+from ethereum.tools import tester
+from ethereum.tools.tester import TransactionFailed
+from pytest import raises, fixture as pytest_fixture
+from utils import garbageBytes20, garbageBytes32, stringToBytes
+
+
+def test_contract_upgrading(localFixture, universe, controller):
```

Comment:
where is universe coming from, I would expect to see a universe pytest_fixture

Reply:
Its in `conftest.py`. There are a lot we pre-make available when using the Kitchen Sink

Reply:
makes sense. looks good